src/Parser/OgParser.php

```diff
@@ -24,64 +24,64 @@ public function parse(string $content): Meta
             $meta->setTitle(htmlspecialchars_decode($matches[1]));
         }
 
-        preg_match('/<meta.*name=\"description\".*content=\"(.+)\"\s*[\/]*\>/Uis', $content, $matches);
+        preg_match('/<meta.*name=\"description\".*content=\"(.+)\"/Uis', $content, $matches);
```

Comment:
why we need this changes?

Reply:
The pattern is too greedy, it grabs additional (probably non standard) attributes after the "content" attribute, for example I encountered the following code:

`<meta name="apple-itunes-app" content="app-id=1282107098" class="next-head" />`

The regex grabs `class="next-head"` also

Reply:
This solution just skips everthing after the content-attribute, there may be better solutions, I have another look

Reply:
looks good, can you please add test for it?

Reply:
I have added a test for non standard attributes after the "content" attribute, all tests including the original ones work